src/Notifynder/Traits/NotifableBasic.php

```diff
@@ -135,4 +135,22 @@ public function getNotifications($limit = null, $order = 'desc')
 
         return $query->get();
     }
+
+    /**
+     * Get all unread Notifications.
+     * 
+     * @param null|int $limit
+     * @param string $order
+     * @return \Illuminate\Database\Eloquent\Collection
+     */
+    public function getNotificationsNotRead($limit = null, $order = 'desc')
```

Comment:
Pls add an unittest for this new method, like: https://github.com/fenos/Notifynder/blob/master/tests/integration/Traits/NotifableTest.php#L77-L89